chart/values.yaml

```diff
@@ -0,0 +1,152 @@
+# Default values for selfservice-api.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+managedByHelm: true
+environment: "develop"
+
+image:
+  repository: 579478677147.dkr.ecr.eu-central-1.amazonaws.com/selfservice-api/app
+  pullPolicy: Always
+  tag: "latest"
```

Comment:
i guess we don't want to just use latest, but a specific version when we do deployments